### PR TITLE
[translate/tts] check reply for args if none provided

### LINKED ISSUE #23: [translate/tts] If no args provided, check message replied to
QoL change: if `context.args` is empty, check for `args` in `message.reply_to_message`.

### PR DESCRIPTION
**Pull request by obviyus**
Fixes #23.

### CODE REVIEW

File: api/translate.py [hunk 1]
```diff
@@ -6,10 +6,17 @@ def translate(update, context):
...
+        try:
+            args = message.reply_to_message.text or message.reply_to_message.caption
+            translator = Translator()
+            translated = translator.translate(args, dest='en')
+            text = translated.text
+        except AttributeError:
+            # blame Udit for this example
+            text = "*Usage:* `/tl {DEST} - {SENTENCE}`\n"\
+                   "*Example:* `/tl en - watashi wa anato no suki desu`\n"\
+                   "Defaults to `en` if none provided."
```
Comment (Tyagdit) [inline api/translate.py]: Mention the reply usage here

**Review by Tyagdit — CHANGES_REQUESTED** [review]

File: api/translate.py [hunk 2]
```diff
@@ -6,10 +6,17 @@ def translate(update, context):
...
+        try:
+            args = message.reply_to_message.text or message.reply_to_message.caption
+            translator = Translator()
+            translated = translator.translate(args, dest='en')
+            text = translated.text
+        except AttributeError:
+            # blame Udit for this example
+            text = "*Usage:* `/tl {DEST} - {SENTENCE}`\n"\
+                   "*Example:* `/tl en - watashi wa anato no suki desu`\n"\
+                   "Reply with `/tl` to a message to translate. Defaults to `en` if none provided."
```
Comment (Tyagdit) [inline api/translate.py]: This makes it sound like you can provide a language when you reply

**Review by Tyagdit — CHANGES_REQUESTED** [review]

File: api/translate.py (hunk 2 above)
Comment (obviyus) [reply api/translate.py]: Could add that, not sure how to right now

Comment (Tyagdit) [reply api/translate.py]: yea just make that known to the user 
```suggestion
                   "Defaults to `en` if none provided."
                   "Reply with `/tl` to a message to translate it to english." 
```

File: api/tts.py [hunk 3]
```diff
@@ -4,13 +4,18 @@
...
-        )
+        try:
+            args = message.reply_to_message.text or message.reply_to_message.caption
+            tts = gTTS(args, lang='ja')
+            message.reply_audio(audio=tts.get_urls()[0])
+        except AttributeError:
+            message.reply_text(
+                text="*Usage:* `/tts {LANG} - {SENTENCE}`\n"
+                     "*Example:* `/tts ru - cyka blyat`\n"
+                     "Reply with `/tts` to a message to convert. Defaults to `ja` if none provided.",
```
Comment (Tyagdit) [inline api/tts.py]: Bro here also

**Review by Tyagdit — CHANGES_REQUESTED** [review]

File: api/tts.py (hunk 3 above)
Comment (Tyagdit) [reply api/tts.py]: Also fix the flake errors

**Review by Tyagdit — APPROVED** [review]

**Pull request merged**